### ISSUE TITLE
HTML in .vue file not lintin w/Beta

### ISSUE THREAD

**Issue by Snugug** [vuejs/eslint-plugin-vue#191]
* **ESLint Version:** 4.7.2
* **eslint-plugin-vue Version:** 3.13.0
* **Node Version:** 8.4.0

**Please show your full configuration:**

```
// http://eslint.org/docs/user-guide/configuring

module.exports = {
  root: true,
  parserOptions: {
    sourceType: 'module',
    parser: 'babel-eslint',
  },
  env: {
    browser: true,
  },
  extends: [
    'airbnb-base',
    'plugin:vue/recommended',
  ],
  // required to lint *.vue files
  plugins: [
    'html',
    'vue',
  ],
  // check if imports actually resolve
  'settings': {
    'import/resolver': {
      'webpack': {
        'config': 'build/webpack.base.conf.js',
      },
    },
  },
  // add your custom rules here
  'rules': {
    // don't require .vue extension when importing
    'import/extensions': ['error', 'always', {
      'js': 'never',
      'vue': 'never',
    }],
    // allow optionalDependencies
    'import/no-extraneous-dependencies': ['error', {
      'optionalDependencies': [
        'test/unit/index.js',
        'generators/**/*.js',
      ],
    }],
    // allow debugger during development
    'no-debugger': process.env.NODE_ENV === 'production' ? 2 : 0,
    // Disable max-length, because we don't really care
    'max-len': 0,
    'vue/valid-v-bind': 2,
  }
}
```

**What did you do? Please include the actual source code causing the issue.**

```vue
<template>
  <div class="hello">
    <h1>{{ msg }}</h1>
    <h2 :aaa>Essential Links</h2> <!-- Expecting this to error in my lint -->
    <ul>
      <li><a href="https://vuejs.org" target="_blank" rel="noopener">Core Docs</a></li>
      <li><a href="https://forum.vuejs.org" target="_blank" rel="noopener">Forum</a></li>
      <li><a href="http://chat.vuejs.org/" target="_blank" rel="noopener">Vue Community Chat</a></li>
      <li><a href="https://twitter.com/vuejs" target="_blank" rel="noopener">Twitter</a></li>
      <li><a href="http://vuejs-templates.github.io/webpack/" target="_blank" rel="noopener">Docs for This Template</a></li>
    </ul>
    <h2>Ecosystem</h2>
    <ul>
      <li><a href="http://router.vuejs.org/" target="_blank" rel="noopener">vue-router</a></li>
      <li><a href="http://vuex.vuejs.org/" target="_blank" rel="noopener">vuex</a></li>
      <li><a href="http://vue-loader.vuejs.org/" target="_blank" rel="noopener">vue-loader</a></li>
      <li><a href="https://github.com/vuejs/awesome-vue" target="_blank" rel="noopener">awesome-vue</a></li>
    </ul>
  </div>
</template>

<script>
export default {
  name: 'hello',
  data() {
    return {
      msg: 'Welcome to Your Vue.js PWA',
    };
  },
};
</script>

<!-- Add "scoped" attribute to limit CSS to this component only -->
<style>
h1, h2 {
  font-weight: normal;
}

ul {
  list-style-type: none;
  padding: 0;
}

li {
  display: inline-block;
  margin: 0 10px;
}

a {
  color: #35495E;
}
</style>
```

**What did you expect to happen?**

I expected to receive an error 

**What actually happened? Please include the actual, raw output from ESLint.**

No output.

**Comment by armano2** [comment]
once more:
https://github.com/BenoitZugmeyer/eslint-plugin-html#htmlhtml-extensions

There is conflict between eslint plugin html and vue,

you have to set list of supported extenstions:
```json
"settings": {
    "html/html-extensions": [".html"],  // consider .html files as HTML
}
````

**Comment by Snugug** [comment]
Thanks, my Googling didn't find that.
